dist/cjs/index.js

```diff
@@ -4599,6 +4477,128 @@ var WebRtcCommunication = /*#__PURE__*/function (_MewConnectCommon) {
   return WebRtcCommunication;
 }(MewConnectCommon);
 
+var name = "@myetherwallet/mewconnect-web-client";
+var homepage = "https://github.com/myetherwallet/MEWconnect-web-client";
+var version = "2.2.0-beta.15";
+var main = "dist/cjs/index.js";
+var module$1 = "dist/esm/index.js";
+var scripts = {
```

Comment:
this seems wrong to me, why include the full package.json in the bundle when all you need is the version... can't u inline the version at build time or sth?